test-app/src/widgets/Draggable.js

```diff
@@ -5,11 +5,12 @@ import MenuItem from './MenuItem';
 import Button from './Button';
 
 export default class Draggable extends createjs.Container {
-  constructor(options, dropArr, tabIndex, callBack) {
+  constructor(options, dropArr, tabIndex, callBack, startContainer) {
```

Comment:
The startContainer is the new element added for maintaining DOM structure. The draggables are children of the drop zones or their startContainer, allowing for the correct tab order